web_src/js/components/RepoActionView.vue

```diff
@@ -386,8 +386,8 @@ export function initRepositoryActionView() {
         <a class="muted" :href="run.commit.link">{{ run.commit.shortSHA }}</a>
         {{ run.commit.localePushedBy }}
         <a class="muted" :href="run.commit.pusher.link">{{ run.commit.pusher.displayName }}</a>
-        <span class="ui label" v-if="run.commit.shortSHA">
-          <a :href="run.commit.branch.link">{{ run.commit.branch.name }}</a>
+        <span class="ui label tw-max-w-full" v-if="run.commit.shortSHA">
+          <a class="gt-ellipsis" :href="run.commit.branch.link">{{ run.commit.branch.name }}</a>
```

Comment:
(as above, here is also a good sample that use `.ui.label .gt-ellipsis`)

Reply:
Will check after https://github.com/go-gitea/gitea/pull/30475 merges.

Reply:
How about take  Improve "label" ellipsis #30479 ahead?

Reply:
Let's see here after that.

Reply:
Hmm no, it looks horrible now

<img width="896" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/cf5ee241-d5a4-4c6b-b7f3-322e09a6d378">

Reply:
Because I saw `ui label tw-max-w-full`, so `max-width: 100%` is not right here?